src/seedu/addressbook/AddressBook.java

```diff
@@ -508,7 +508,7 @@ private static String executeDeletePerson(String commandArgs) {
             return MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
         }
         final String[] targetInModel = getPersonByLastVisibleIndex(targetVisibleIndex);
-        return deletePersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
+        return hasDeletedPersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
```

Comment:
This may not be an appropriate change :-). The method perform deletion and the return value indicates whether the deletion is successful or not. So, it is a little strange to call it "hasDeletedPerson....".